services/productService.js

```diff
@@ -0,0 +1,19 @@
+const { productDao,reviewDao } = require("../models");
+
+const detailProduct = async ( productId ) => {
+    const product = await productDao.getProductById( productId );
+    if ( !product ) {
+        const err = new Error("INVALID_PRODUCT");
+        err.statusCode = 406;
+        throw err;
+    }
+    const detailProduct = await productDao.getProductDetail( productId );
+    const productReview = await reviewDao.getReviewByProduct( productId );
+    const productImage = await productDao.getProductImage( productId );
```

Comment:
반환되는 값이 여러개인지 여부를 반드시 코드에 명시해주는 것이 좋습니다.
아래와 같이 배열로 반환되는 데이터는 복수로 변수명을 설정해주세요~
```suggestion
    const reviews = await reviewDao.getReviewByProduct( productId );
    const images = await productDao.getProductImage( productId );
```

Reply:
넵 수정 완료했습니다!